festim/hydrogen_transport_problem.py

```diff
@@ -808,6 +808,13 @@ def post_processing(self):
                 # if filename given write export data to file
                 if export.filename is not None:
                     export.write(t=float(self.t))
+            elif isinstance(export, F.VolumeQuantity):
+                export.compute(self.dx)
+                # update export data
+                export.t.append(float(self.t))
 
+                # if filename given write export data to file
+                if export.filename is not None:
+                    export.write(t=float(self.t))
```

Comment:
Could test this

Reply:
So I think this is gonna be refactored when we start adding more derived quantities anyway, which is why I didn't bother testing it here